modules/primer-marketing-support/lib/variables.scss

```diff
@@ -1,3 +1,13 @@
 // Type
 $alt-body-font: Roboto, -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", sans-serif;
 $alt-mono-font: $mono-font;
```

Comment:
hmm should I move these variables to the `primer-marketing-type` module? @sophshep @broccolini